Antonio_Ramos_Documentation

```diff
@@ -140,12 +148,86 @@ new code - pFC = r"G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\CARR\California\Ba
+
+- ZCTA_GoodLabs_Trans.py changes the CRS for GoodLabs.shp and exports GoodLabs_New.shp 
+
+- ZCTA_GoodLabs_Trans.py also also reads a csv file (DEC_00_SF3_DP3_with_ann.csv) and joins it with an imported shapefile (tl_2010_06_zcta500.shp) changes the CRS and then exports CA_ZCTA_Man.shp
+
+- all other R and python scripts do not seem to be relevant to the data that is currently being used by the K-function Local_3Stage.py script. However, it is not clear where the data file CA_Block_New.shp came from which is the third data file being read into the K-function Local_3Stage.py script. 
+
+- In the folder located at "G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\Block Level Analysis" I found two programs that might be related to CA_Block_New.shp 
+ 1. CA_Block_Trans.py
+ 2. CA_WAC_Trans.py 
+both programs are written August 8th but there is no documentation for why 
+
+- CA_Block_Trans.py has a note that says "Need to figure out how to change shapefile from ANSI 1252 to UTF-8 encoding." Not sure if this was ever done will look for more up to date file. After examining the code it looks like CA_Block_Trans.py is the same as CA_WAC_Trans.py except CA_Block_Trans.py changes the geometry and CRS. CA_Block_Trans.py reads in a csv.gz file (ca_wac_S000_JT00_2002.csv.gz) and and joins it with an imported shapefile (CA_block_2000.shp) then changes the geometry and CRS and exports a shapefile (CA_2000_Tot_Emp.shp) however, this is not the block data file which is being read into K-function Local_3Stage.py. 
+
+- found the origin of ca_wac_S000_JT00_2002.csv.gz, https://lehd.ces.census.gov/data/lodes/LODES7/ca/wac/  (ask prof. buzard if she can help clarify what the content of this file is) 
+
+06/13/2018
+- returning to the issue GoodLabs_New.shp not plotting anything when added as a layer to a blank ArcMap map, when I tested GoodLabs.shp the original file, the points do appear plotted on the map. 
+
+-changed projected coordinate system (PCS) for GoodLabs.shp in ArcGis to make it compatibale with the PCS of CA_Block_New (NAD 1983 Contiguous USA Albers) the outputted file with the updated PCS is GoodLabs_Project.shp located with the above mentioned files at "G:\MAX-Filer\Research\Labs-kbuzard-S18\Admin\CARR\California\Base Data" (do you want me to outline the steps for changing the PCS in ArcGis?) 
+
+- at this point both GoodLabs.shp and CA_Block_New.shp open and are plotted correctly in arcmap. the zip code data file hoewver is not projecting properly in arcmap. 
+
+-having a lot of trouble being able to remake the zipcode with manufacturing employment shapefile. When I add the manufa emp data in python then I have issues with the CRS in arcgis where either
+1. i change the CRS in python but then it doesnt project the data properly in arcgis
+2. i do only the join in python and try to change the CRS in arcgis, once I change the CRS again the data ceases to be projected 
+3. i try to do the join in arcgis and then attempt to change CRS, can't even do the join
```

Comment:
I'm not sure if this means you don't know how to do the join, or if you're having problems executing it. Either way, I can help troubleshoot if you need.

So both the base zip code file and the one with zipcode level employment are shapefiles? If so, have you tried manipulating the geographic variables before you merge them together? That might avoid any conflict from the two being different

Reply:
They arent both shapefiles. One is an excel sheet and the other is a shapefile, but I managed to do the join and create a file that displays properly on arcgis along with the labs and the block data. However, as mentioned in the previous comment, the python program is still not running correctly. So my hunch, based on your comment, is that it is the CRS. I did more research and realized that Will's code as written is only changing the PCS metadata, but now I am having trouble writing code that does change the CRS and geometry. I will update you in the next pull request.